packages/devtools_app/lib/src/flutter/scaffold.dart

```diff
@@ -190,21 +209,27 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
         ),
     ];
 
-    return DragAndDrop(
-      handleDrop: _importController.importData,
-      child: AnimatedBuilder(
-        animation: appBarCurve,
-        builder: (context, child) {
-          return Scaffold(
-            appBar: _buildAppBar(),
-            body: child,
-            bottomNavigationBar: _buildStatusLine(context),
-          );
-        },
-        child: TabBarView(
-          physics: const NeverScrollableScrollPhysics(),
-          controller: _tabController,
-          children: tabBodies,
+    return StreamProvider<Screen>(
```

Comment:
ValueListenableBuilder would provide the same functionality without requiring you to specify the initialData.